From fb411117471adf6822b04b9115fb4c52da140f01 Mon Sep 17 00:00:00 2001
From: Yaroslav Lavrinenko <y.lavrinenko@samsung.com>
Date: Thu, 8 Sep 2022 16:10:08 +0300
Subject: [PATCH 05/27] [device] Add device memory mapping for SGX mode

We introduce device memory mapping in the same way as for direct mode.

Resolve: AXDIMM-381
---
 application/axdimm_secndp_test/Makefile               |  2 --
 .../axdimm_secndp_test/secndp.manifest.template       | 11 ++++-------
 2 files changed, 4 insertions(+), 9 deletions(-)

diff --git a/application/axdimm_secndp_test/Makefile b/application/axdimm_secndp_test/Makefile
index f7a44e84..a359104e 100644
--- a/application/axdimm_secndp_test/Makefile
+++ b/application/axdimm_secndp_test/Makefile
@@ -1,5 +1,3 @@
-# PyTorch and the pre-trained model must be installed on the system. See README for details.
-
 ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
 
 ifeq ($(DEBUG),1)
diff --git a/application/axdimm_secndp_test/secndp.manifest.template b/application/axdimm_secndp_test/secndp.manifest.template
index dcfefc9b..043e5cbc 100644
--- a/application/axdimm_secndp_test/secndp.manifest.template
+++ b/application/axdimm_secndp_test/secndp.manifest.template
@@ -1,4 +1,4 @@
-# PyTorch manifest template
+# secndp axdimm manifest template
 
 loader.entrypoint = "file:{{ gramine.libos }}"
 libos.entrypoint = "{{ entrypoint }}"
@@ -27,7 +27,7 @@ fs.mounts = [
 ]
 
 sgx.nonpie_binary = true
-sgx.enclave_size = "4G"
+sgx.enclave_size = "1G"
 sgx.thread_num = 512
 sgx.debug = true
 sgx.insecure__enable_shared_range = "true"
@@ -43,11 +43,8 @@ sgx.trusted_files = [
   "file:{{ axdimm_driver }}/build/secure/",
 ]
 
-sgx.allowed_files = [
-  "file:run_kaggle_pt/",
-  "dev:/dev/axdimm_meminfo",
-  "dev:/dev/axdimm_device0"
-]
+#sgx.allowed_files = [
+#]
 
 
 # sgx.enable_stats = true
-- 
2.34.1

